Tests/SessionManagerTests/SessionManagementTest.swift

```diff
@@ -17,15 +17,15 @@ final class SessionManagementTest: XCTestCase {
 
     func test_createSessionID() async throws {
         let sessionId = try SessionManager.generateRandomSessionID();
-        let session = SessionManager(sessionId: sessionId)
+        let session = SessionManager(sessionId: sessionId, sessionNamespace: "sfa")
```

Comment:
We'd need to test for the case in which sessionNamespace is nil here too.

Reply:
Did that. Also passing with namespace.